elliottlib/cli/find_bugs_wo_target_version.py

```diff
@@ -0,0 +1,54 @@
+import json
+import click
+from elliottlib import (Runtime, logutil)
+from elliottlib.cli import common
+from elliottlib.cli.find_bugs_sweep_cli import FindBugsSweep, print_report
+
+logger = logutil.getLogger(__name__)
+
+
+@common.cli.command("find-bugs:notv", short_help="Find qualified ART managed jira bugs (sweep) that do not have a "
+                                                 "target version field set")
+@click.option("--comment", "comment",
+              required=False,
+              help="Add comment to found bugs")
+@click.option("--report",
+              required=False,
+              is_flag=True,
+              help="Output a detailed report of found bugs")
+@click.pass_obj
+@common.click_coroutine
+async def find_bugs_no_tv(runtime: Runtime, comment, report):
```

Comment:
I know what this PR is about, and still i had to spend three seconds imagining what `television` has to do with bugs. Can reduce unnecessary abbreviations?

Reply:
That was my "bad (horrible) naming" day, I'll change it